lib/membrane_http_adaptive_stream/sink.ex

```diff
@@ -101,17 +111,20 @@ defmodule Membrane.HTTPAdaptiveStream.Sink do
   end
 
   @impl true
-  def handle_caps(Pad.ref(:input, track_id), %CMAF.Track{} = caps, _ctx, state) do
+  def handle_caps(Pad.ref(:input, track_id) = pad_ref, %CMAF.Track{} = caps, ctx, state) do
     {header_name, manifest} =
       if Manifest.has_track?(state.manifest, track_id) do
         # Arrival of new caps for an already existing track indicate that stream parameters have changed.
         # According to section 4.3.2.3 of RFC 8216, discontinuity needs to be signaled and new header supplied.
         Manifest.discontinue_track(state.manifest, track_id)
       else
+        track_name = ctx.pads[pad_ref].options[:track_name] || track_id |> parse_track_name()
```

Comment:
This line is not clear. In fact, I had to look at `parse_track_name/1` to understand that it should be read as `(ctx.pads[pad_ref].options[:track_name] || track_id) |> parse_track_name()` instead of `ctx.pads[pad_ref].options[:track_name] || (track_id |> parse_track_name())` which was my original interpretation. I think it would be better without pipe operator

```suggestion
        track_name = parse_track_name(ctx.pads[pad_ref].options[:track_name] || track_id)
```